src/main/java/com/softserve/teamproject/entity/Location.java

```diff
@@ -28,19 +28,21 @@
   private String name;
 
   @ManyToOne
+  @JoinColumn(name = "coordinator_id", referencedColumnName = "id")
+  private User coordinator;
+
+  @ManyToOne
   @JoinColumn(name = "country_id", referencedColumnName = "id", nullable = false)
   private Country country;
 
-  @OneToOne
-  @JoinColumn(name = "coordinator_id", referencedColumnName = "id", nullable = false)
-  private User coordinator;
```

Comment:
I dont know for sure, but i can change if it nessesary .  
I thought that could be 2 coordinators in one location so thats why.